airflow/hooks/hive_hooks.py

```diff
@@ -46,29 +52,117 @@ class HiveCliHook(BaseHook):
 
     The extra connection parameter ``auth`` gets passed as in the ``jdbc``
     connection string as is.
+
+    :param mapred_queue: queue used by the Hadoop CapacityScheduler
+    :type  mapred_queue: string
+    :param mapred_queue_priority: priority within CapacityScheduler queue.
```

Comment:
It should, see for instance : https://hadoop.apache.org/docs/r2.7.1/hadoop-yarn/hadoop-yarn-site/FairScheduler.html
I changed the docstring to reflect this.